app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PostsHelper
```

Comment:
What is this file doing?

Reply:
Will remove it. It is not providing any functionality yet